azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+# ASP.NET Core (.NET Framework)
```

Comment:
use please pipeline such as LoanPipelines nuget pipeline

Reply:
[Here is the example](https://github.com/Interfirst/Interfirst.Skynet.LoanPipelines/blob/master/azure-pipelines-nuget.yml)